From ddda95657f634aa6261f55bde530b33995411431 Mon Sep 17 00:00:00 2001
From: julien delnatte <julien.delnatte@gmail.com>
Date: Wed, 30 Jun 2021 16:04:39 +0200
Subject: [PATCH] binding

---
 binding.gyp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/binding.gyp b/binding.gyp
index 340bc600..389ec9cb 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -1,6 +1,6 @@
 {
     "targets": [{
-        "target_name": "addon",
+        "target_name": "gridapp",
         "cflags!": [ "-fexceptions", "--max-old-space-size=3000" ],
         "cflags_cc!": [ "-fexceptions", "-fno-rtti", "-fno-exceptions" ],
         "sources": [
@@ -36,7 +36,7 @@
           'GCC_ENABLE_CPP_RTTI': 'YES',
         }]
       ],
-      'defines': [ 
+      'defines': [
         'NAPI_CPP_EXCEPTIONS',
         'TASKING_INTERNAL'
       ]
-- 
2.20.1 (Apple Git-117)

